node/tests/SharedTests.ts

```diff
@@ -12240,6 +12240,159 @@ export function runBaseTests(config: {
         },
         config.timeout,
     );
+
+    describe.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        "update_connection_password_%p",
+        (protocol) => {
+            const NEW_PASSWORD = "new_password";
+            const WRONG_PASSWORD = "wrong_password";
+            /**
+             * Test replacing the connection password without immediate re-authentication.
+             * Verifies that:
+             * 1. The client can update its internal password
+             * 2. The client remains connected with current auth
+             * 3. The client can reconnect using the new password after server password change
+             * Currently, this test is only supported for cluster mode,
+             * since standalone mode dont have retry mechanism.
+             */
+            it("test_update_connection_password", async () => {
+                await runTest(async (client: BaseClient) => {
+                    try {
+                        if (client instanceof GlideClient) {
+                            return;
+                        }
+
+                        const result = await client.updateConnectionPassword(
+                            NEW_PASSWORD,
+                            false,
+                        );
+                        expect(result).toEqual("OK");
+
+                        await client.set("test_key", "test_value");
+                        const value = await client.get("test_key");
+                        expect(value).toEqual("test_value");
+                        await expect(
+                            client.configSet({
+                                requirepass: NEW_PASSWORD,
+                            }),
+                        ).resolves.toBe("OK");
+                        await client.customCommand([
```

Comment:
`CLIENT KILL` doesn't kill the client which sends that. You need to test with 2 clients.

Reply:
Yes you right, theres skipme param, which i thoght its default is false, apparently its true. so i just need to set it to false.